backend/manager/modules/utils/src/main/java/org/ovirt/engine/core/utils/ovf/OvfReaderWriterUtils.java

```diff
@@ -0,0 +1,41 @@
+package org.ovirt.engine.core.utils.ovf;
+
+import org.ovirt.engine.core.common.businessentities.VmDeviceGeneralType;
+import org.ovirt.engine.core.common.utils.VmDeviceType;
+
+public abstract class OvfReaderWriterUtils {
```

Comment:
Why making a class abstract, if it contains only static methods?

Reply:
Because I had originally played with the idea to make it a base class and then changed my mind and forgot to update the declarations :-). Done.